js/class.js

```diff
@@ -0,0 +1,42 @@
+export default class Liberary {
+  constructor(books) {
+    this.books = books;
+  }
+
+    fetchBooks = () => {
+      let BooksCards = [];
+      if (this.books) {
+        BooksCards = this.books.map(
+          (book) => `<div class='${book.title} book-container'>
+            <h1 class="book-title">" ${book.title} " <span class="book-author">by ${book.author}</span></h1>
+            <button id='${book.title}' class='btn-remove' onclick="removeBook('${book.title}')">
+            remove
+            </button>
+            </div><hr>`,
+        );
+      }
+      return BooksCards;
+    };
+
+    addBook = (obj) => {
+      if (obj.title !== '' && obj.author !== '') {
+        this.books.push(obj);
+        localStorage.setItem('books', JSON.stringify(this.books));
+      }
+    };
+
+    removeBook = (title) => {
+      const updatedList = this.books.filter((book) => book.title !== title);
+      this.books = updatedList;
+      localStorage.setItem('books', JSON.stringify(updatedList));
+    };
```

Comment:
- [ ] It will be great to remove the book based on an `ID` the reason behind this is that you are removing the book based on the title... Although this is fine, it presents a problem. Picture this scenario, What if we have two books with the same title? This means it will remove both books which is not the intention. 

### HINT:

- When creating a book, have an id property that increases by one every time a new book is created. This will ensure that you have a bookID that you can use to filter your books with when implementing the remove method